save-demo/src/main/kotlin/com/saveourtool/save/demo/utils/KubernetesUtils.kt

```diff
@@ -0,0 +1,136 @@
+/**
+ * Utils for kubernetes client
```

Comment:
why you extract these methods to util class? I guess KubernetesService is good place for them

Reply:
I would like `KubernetesService` to look more pretty and more "high-level"

Reply:
now logic split to two places, extension is extension, if you want to have it in a separate place -- create a class for it

Reply:
Discussed offline - decided to postpone until `demo`/`orchestrator-common` refactoring